recipe/install_llvm.sh

```diff
@@ -23,7 +23,6 @@ if [[ "${PKG_NAME}" == libllvm-c* ]]; then
 elif [[ "${PKG_NAME}" == libllvm* ]]; then
     cmake --install ./build --prefix=./temp_prefix
     # all other libraries
-    mv ./temp_prefix/lib/libLLVM-${SOVER_EXT}${SHLIB_EXT} $PREFIX/lib
```

Comment:
This follows https://github.com/llvm/llvm-project/commit/91a384621e5b762d9c173ffd247cfeadd5f436a2; the symlink still, but - at least as of rc4 - exists but has a wrong configuration (so the file-check fails because the symlink is dead).

There's a comment to that effect also in the PR associated with that commit.